### PR TITLE
Update add-your-organization-brand-to-encrypted-messages.md

### PR DESCRIPTION
**Pull request by samschan-ms**
Added clarification the branding applies to mail sent from your organizaiton's online mailboxes. @KCCross  to review

### CODE REVIEW

File: microsoft-365/compliance/add-your-organization-brand-to-encrypted-messages.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ If you'd like more control, use Office 365 Advanced Message Encryption to create
 - Whether you want to allow emails to be revoked
 - Whether you want emails sent to external recipients to expire after a specified number of days.
 
-Once you've created the templates, you can apply them to encrypted emails by using Exchange mail flow rules. If you have Office 365 Advanced Message Encryption, you can revoke any email that you've branded by using these templates.
+Once you've created the templates, you can apply them to encrypted emails sent from by using Exchange mail flow rules. If you have Office 365 Advanced Message Encryption, you can revoke any email that you've branded by using these templates.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/add-your-organization-brand-to-encrypted-messages.md]:
```suggestion
Once you've created the templates, you can apply them to encrypted emails sent from your online mailbox by using Exchange mail flow rules. If you have Office 365 Advanced Message Encryption, you can revoke any email that you've branded by using these templates.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested change if you agree with it. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Comment by KCCross** [comment]
Added content through a PR in the private repo. Closing this PR as the changes were incorporated elsewhere.

**Comment by KCCross** [comment]
https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/17000

**Pull request closed without merging**